useful-scripts/misc-test-utils/src/index.ts

```diff
@@ -28,24 +39,134 @@ const start = async () => {
         await createPublicationAndRelease();
         break;
 
+      case 'create new publication':
+        await createPublication();
+        break;
+
       case 'publish a new release':
         await createReleaseAndPublish();
         break;
 
       case 'upload subject':
-        // eslint-disable-next-line no-case-declarations
         const release = await inquirer.prompt({
           name: 'id',
           type: 'input',
           message: 'Release ID from existing publication',
           prefix: '>',
         });
 
-        await uploadSingleSubject(release.id);
+        const numOfSubjects = await inquirer.prompt({
+          name: 'number',
+          type: 'number',
+          message: 'How many subjects do you want to upload?',
+          prefix: '>',
+          default: 1,
+        });
+
+        const fast = await inquirer.prompt({
+          name: 'shouldBeFast',
+          type: 'confirm',
+          message:
+            "would you like the uploadSubject function to exit as soon as a 'QUEUED' status is received? (this can be useful for uploading lots of subjects quickly)",
+        });
+
+        for (let i = 0; i < numOfSubjects.number; i++) {
+          await uploadSingleSubject(release.id, fast.shouldBeFast);
+        }
+        break;
+
+      case 'create new methodology':
+        const publication = await inquirer.prompt({
+          name: 'id',
+          type: 'input',
+          message: 'publication ID from existing publication',
+          prefix: '>',
+        });
+        await createMethodology(publication.id);
+        break;
+
+      case 'generate lorem ipsum content text block (methodology & release)':
+        const methodologyOrRelease = await inquirer.prompt({
```

Comment:
I'm liking the look of this inquirer :)

Reply:
Yeah, it's a pretty neat library. Initially got the idea from <a href="https://github.com/benawad/tsconfig.json">this project</a>